xdsl/interpreters/onnx.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, cast
+
+from xdsl.dialects import onnx
+from xdsl.interpreter import (
+    Interpreter,
+    InterpreterFunctions,
+    impl,
+    register_impls,
+)
+from xdsl.interpreters.shaped_array import ShapedArray
+
+try:
+    import numpy as np  # pyright: ignore
+except (ImportError, ModuleNotFoundError) as exc:
+    print(exc)
```

Comment:
```suggestion
import numpy as np
```
Please add this file to the list of files in pyproject.toml that only shows pyright errors if it's open